code/modules/economy/requisition/requisition_contracts.dm

```diff
@@ -252,40 +287,78 @@ ABSTRACT_TYPE(/datum/rc_entry/seed)
 		src.gene_factors = length(src.gene_reqs)
 		..()
 
+	// override to account for any item type that has plantgenes and planttype. Boy do I miss interfaces.
+	proc/GetGenesAndPlantType(atom/eval_item)
+		if (!istype(eval_item,/obj/item/reagent_containers/food/snacks/plant)) return
+		var/GenesAndType = list()
+		var/obj/item/reagent_containers/food/snacks/plant/plant = eval_item
+		GenesAndType += plant.plantgenes
+		GenesAndType += plant.planttype
+		return GenesAndType
+
 	rc_eval(atom/eval_item)
 		. = ..()
-		if(rollcount >= count) return // Standard skip-if-complete
-		if(!istype(eval_item,/obj/item/seed)) return // Not a seed? Skip it
-
-		var/obj/item/seed/cultivar = eval_item
-		if(!cultivar.plantgenes) return // No genome? Skip it
-		if(cultivar.planttype.name != cropname) return // Wrong species? Skip it
+		if(rollcount >= count) return // Standard skip-if-complet
+		var/genesandtype = GetGenesAndPlantType(eval_item)
+		if (!genesandtype) return
+		var/datum/plantgenes/genes
+		var/datum/plant/plant_type
+		genes = genesandtype[1]
+		plant_type = genesandtype[2]
+
+		if(!genes) return // No genome? Skip it
+		if(plant_type.name != cropname) return // Wrong species? Skip it
 		if(!extra_eval(eval_item)) return
 
 		gene_count = 0
 		for(var/index in gene_reqs) // Iterate over each parameter to see if the genome meets it, or exceeds it in the right direction
 			switch(index)
 				if("Maturation")
-					if(cultivar.plantgenes.growtime >= gene_reqs["Maturation"]) gene_count++
+					if(genes.growtime >= gene_reqs["Maturation"]) gene_count++
 				if("Production")
-					if(cultivar.plantgenes.harvtime >= gene_reqs["Production"]) gene_count++
+					if(genes.harvtime >= gene_reqs["Production"]) gene_count++
 				if("Lifespan")
-					if(cultivar.plantgenes.harvests >= gene_reqs["Lifespan"]) gene_count++
+					if(genes.harvests >= gene_reqs["Lifespan"]) gene_count++
 				if("Yield")
-					if(cultivar.plantgenes.cropsize >= gene_reqs["Yield"]) gene_count++
+					if(genes.cropsize >= gene_reqs["Yield"]) gene_count++
 				if("Potency")
-					if(cultivar.plantgenes.potency >= gene_reqs["Potency"]) gene_count++
+					if(genes.potency >= gene_reqs["Potency"]) gene_count++
 				if("Endurance")
-					if(cultivar.plantgenes.endurance >= gene_reqs["Endurance"]) gene_count++
+					if(genes.endurance >= gene_reqs["Endurance"]) gene_count++
 
 		if(gene_count >= gene_factors) // Compare satisfied parameter count to number of parameters. Met or exceeded means seed satisfies requirements
 			src.rollcount++
```

Comment:
Does this account for seed packets? If not, I'd *suspect* it would be as simple as adding the packet's "amount" variable to rollcount (if applicable) instead of incrementing it by 1.

Reply:
More or less, seed packets aren't technically a stack of items for the same reason seeds haven't been stackable in the past, but it's just an extra cast. 

It should be accounting for seed packets though, finally.